docs/resources/aws_kms_key.md

```diff
@@ -0,0 +1,172 @@
+---
+title: About the aws_kms_key Resource
+---
+
+# aws_kms_key
+
+Use the `aws_kms_key` InSpec audit resource to test properties of a single AWS KMS Key.
+
+AWS Key Management Service (AWS KMS) is a managed service that makes it easy for you to create and control the encryption keys used to encrypt your data. AWS KMS lets you create master keys that can never be exported from the service and which can be used to encrypt and decrypt data based on policies you define.
+
+Each AWS KMS Key is uniquely identified by its key-id or key-arn.
+
+<br>
+
+## Syntax
+
+An aws_kms_key resource block identifies a key by key_arn.
+
+    # Find a kms key by arn
```

Comment:
This does not match what the implementation is doing: the implementation is actually passing the key_id.  If the SDK accepts either one, we should clarify that.